packages/web/app/src/components/organization/billing/InvoicesList.tsx

```diff
@@ -7,7 +7,7 @@ import { CurrencyFormatter } from './helpers';
 export const InvoicesList: React.FC<{
   organization: OrganizationFieldsFragment & OrgBillingInfoFieldsFragment;
 }> = ({ organization }) => {
-  if (organization.billingConfiguration?.invoices?.length === 0) {
+  if (null == organization.billingConfiguration?.invoices?.length) {
```

Comment:
good reason to enable https://eslint.org/docs/latest/rules/yoda rule in our config @dotansimha @kamilkisiela 
```suggestion
  if (organization.billingConfiguration?.invoices?.length == null) {
```

Reply:
xD